iotdb-core/datanode/src/main/java/org/apache/iotdb/db/consensus/SchemaRegionConsensusImpl.java

```diff
@@ -46,6 +47,10 @@ private SchemaRegionConsensusImpl() {
     // do nothing
   }
 
+  public static boolean isRatis() {
+    return SchemaRegionConsensusImplHolder.CONF.getSchemaRegionConsensusProtocolClass().equals(ConsensusFactory.RATIS_CONSENSUS);
```

Comment:
ConsensusFactory.RATIS_CONSENSUS.equals(SchemaRegionConsensusImplHolder.CONF.getSchemaRegionConsensusProtocolClass())

Reply:
Dan man you can really write good code bro m3?